tensorflow/lite/micro/kernels/svdf.cc

```diff
@@ -52,284 +41,12 @@ constexpr int kOutputTensor = 0;
  * and resizes the output tensor. Micro runtime does not support tensor
  * resizing.
  */
```

Comment:
move the comment to svdf_common.cc as well

Reply:
Done